src/utils/styleVariables.js

```diff
@@ -58,13 +65,25 @@ export default {
   size_large: pxToEm(40),
   size_jumbo: pxToEm(52),
 
-  spacing_quarter: pxToEm(2),
-  spacing_half: pxToEm(4),
-  spacing_single: pxToEm(8),
-  spacing_oneAndAHalf: pxToEm(12),
-  spacing_double: pxToEm(16),
-  spacing_triple: pxToEm(24),
-  spacing_quad: pxToEm(32),
```

Comment:
how might we deprecate these vars, so that they don't disappear all at once?

Reply:
[I tried!](https://github.com/mineral-ui/mineral-ui/commit/d6a4b93dca52917c5d7004bbed2391f1b3fb0ca2#diff-7124768e5148154ecec2ba1c2f92e9ff)

Turns out, deprecating an object property is a lot more difficult than deprecating a function. I got _almost_ everything working [on this branch](https://github.com/mineral-ui/mineral-ui/blob/deprecate-theme-variables/src/utils/styleVariables.js):

- Mapping from, e.g. `spacing_single` ➡️  `space_inline_sm`, `space_inset_sm`, or `space_stack_sm`
- Deprecation message printed correctly to console as a warning
- Each deprecation message only printed once per page load

However, it would print even if none of the deprecated properties were referenced in a component. This is because I couldn't find a way to differentiate between accessing the entire object and accessing a specific property. 😕

Reply:
I think that this type of change is better documented in the changelog without trying to over-engineer it.